src/main/scala/abalone/format/FEN.scala

```diff
@@ -1,33 +1,60 @@
 package strategygames.abalone.format
 
 import strategygames.Player
+import strategygames.abalone.File
+import strategygames.abalone.Piece
 import strategygames.abalone.PieceMap
+import strategygames.abalone.Pos
+import strategygames.abalone.Stone
```

Comment:
You can group these imports like
`strategygames.abalone.{ File, Piece, PieceMap, Pos, Stone }`

Reply:
indeed, and I know it :)
is there a command I can invoke to take care of that so it remains clean even if I forgot ?